docs/website/mkdocs.yml

```diff
@@ -100,6 +100,7 @@ nav:
       - TensorFlow Lite: 'ml-frameworks/tensorflow-lite.md'
       - JAX: 'ml-frameworks/jax.md'
   - 'Deployment configurations':
+      - CPU - Bare-Metal: 'deployment-configurations/bare-metal.md'
       - CPU - Dylib: 'deployment-configurations/cpu-dylib.md'
```

Comment:
The ordering used here will be reflected in the nav:

![image](https://user-images.githubusercontent.com/4010439/125491364-f0cec382-3db5-4719-816d-2eca11856a1a.png)

Let's switch these so dylib is first, since the bare metal doc references to the dylib doc.

```suggestion
      - CPU - Dylib: 'deployment-configurations/cpu-dylib.md'
      - CPU - Bare-Metal: 'deployment-configurations/bare-metal.md'
```

Reply:
Done. Was thinking about the alphabetical order, but definitely more sense with dyllib up front.